uniffi_bindgen/src/interface/mod.rs

```diff
@@ -218,6 +218,42 @@ impl ComponentInterface {
         self.errors.get(name)
     }
 
+    /// Get an Enum definition by name or panic
+    pub fn get_enum_definition_unchecked(&self, name: &str) -> &Enum {
```

Comment:
"unchecked" is generally used for unsafe functions that have undefined behavior in the unexpected case. But maybe it's fine to use the same naming here.